File: python/C_F.py
# Function to convert Celsius to Fahrenheit
def celsius_to_fahrenheit(celsius):
    return (celsius * 9/5) + 32

# Input: Enter temperature in Celsius
celsius = float(input("Enter temperature in Celsius: "))

# Convert to Fahrenheit
fahrenheit = celsius_to_fahrenheit(celsius)

# Output: Print the temperature in Fahrenheit
print(f"{celsius} Celsius is equal to {fahrenheit:.2f} Fahrenheit")

File: C/C_F.c
#include <stdio.h>

// Function to convert Celsius to Fahrenheit
double celsius_to_fahrenheit(double celsius) {
    return (celsius * 9/5) + 32;
}

int main() {
    double celsius, fahrenheit;

    // Input: Enter temperature in Celsius
    printf("Enter temperature in Celsius: ");
    scanf("%lf", &celsius);

    // Convert to Fahrenheit
    fahrenheit = celsius_to_fahrenheit(celsius);

    // Output: Print the temperature in Fahrenheit
    printf("%.2f Celsius is equal to %.2f Fahrenheit\n", celsius, fahrenheit);

    return 0;
}
